ads-service/bootstrap.py

```diff
@@ -4,13 +4,14 @@
 import os
 DB_USERNAME = os.environ['POSTGRES_USER']
 DB_PASSWORD = os.environ['POSTGRES_PASSWORD']
+DB_URL = os.environ['POSTGRES_URL']
 
 
 def create_app():
     """Create a Flask application"""
     app = Flask(__name__)
     #app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///app.db'
-    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://' + DB_USERNAME + ':' + DB_PASSWORD + '@' + 'db/' + DB_USERNAME
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://' + DB_USERNAME + ':' + DB_PASSWORD + '@' + DB_URL + '/' + DB_USERNAME
```

Comment:
I think URL will be a bit misleading since I'd think of a database connection string URI. Would suggest changing this to HOST instead which is more accurate given the implementation. See https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING